recipes/mage-ai/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "mage-ai" %}
+{% set version = "0.4.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6d99fbeb720db9b3d6b643e4197bb6b07463c40e03a4bc750b201ac9162f7eda
+
+build:
+  skip: True  # [win]
+  skip: True  # [py<37]
+  skip_compile_pyc:
+    - "*/templates/*.py"          # These should not (and cannot) be compiled
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Comment:
```suggestion
  number: 0
  entry_points:
    - mage = mage_ai.cli.main:main
```

Reply:
You need to add the entry point. Check the metadata if this is correct.

Reply:
Yes, I added the entry point already.

Reply:
Ah. You did that manually instead of via this comment. Confusing but OK.